src/components/ux-flex-video/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+  "data": {
+    "allowfullscreen": "Allow full screen options.  e.g. allowfullscreen webkitAllowFullScreen mozallowfullscreen",
+    "class": "CSS class customisations",
+    "frameborder": "Is the frameborder of the iframe",
+    "height": "Is the height of the iframe",
+    "src": "Is the source URL for the video. e.g. //www.youtube.com/embed/aiBt44rrslw",
+    "width": "Is the width of the iframe"
```

Comment:
Completely up to you on this. What about removing `Is the` from these four? Still reads well without